guillotina_ratelimit/tests/test_ratelimit.py

```diff
@@ -0,0 +1,81 @@
+from guillotina_ratelimit.tests.package import SERVICE_RATE_LIMITS
+import aiotask_context
+import json
+import asyncio
+
+
+async def test_global_rate_limits(container_requester, dummy_request,
+                                  state_manager, global_rate_limits):
+    aiotask_context.set('request', dummy_request)
+
+    retry_after = None
+    async with container_requester as requester:
+        for i in range(global_rate_limits['hits'] + 1):
+            resp, status = await requester('GET', '/db/guillotina')
+            if status not in (200, 201):
+                # Check status and content
+                assert status == 429
+                assert resp['reason'] == 'Global rate-limits exceeded'
+                retry_after = resp['Retry-After']
```

Comment:
Is it possible to check the headers from the `container_requester` responses?

Reply:
Yup, see https://github.com/plone/guillotina/blob/master/guillotina/tests/test_api.py#L705